src/Pickles/Pickles.ObjectModel/ObjectModel/TableRow.cs

```diff
@@ -45,6 +45,22 @@ public List<string> Cells
             get { return this.tableCells; }
         }
 
+    }
+
+    public class TestTableRow : TableRow
```

Comment:
`TestTableRow` sounds like it's a table for testing purposes. I prefer a name like "TableRowWithTestResult" or something like that.